core/Plugin/MetadataLoader.php

```diff
@@ -49,9 +49,17 @@ public function __construct($pluginName)
      */
     public function load()
     {
+        $defaults = $this->getDefaultPluginInformation();
+        $plugin   = $this->loadPluginInfoJson();
+
+        // use translated plugin description if available
+        if ($defaults['description'] != Piwik::translate($defaults['description'])) {
+            unset($plugin['description']);
```

Comment:
It took me a while to figure out why not `$plugin['description']=Piwik::translate(...)` but I got it... the correct language might be not loaded yet. This way we make sure to translate on demand when requested. I was wondering if this logic was better done in `core\Plugin` but we can still move it later if needed.